udata/core/user/models.py

```diff
@@ -135,11 +135,13 @@ def resources_availability(self):
                 *[org.check_availability() for org in self.organizations]
             )
         )
+        # Filter out the unknown
+        availabilities = [a for a in availabilities if type(a) is bool]
```

Comment:
Why not using `isinstance`?

Reply:
I like that when checking for a basic type because it reads as en English sentence.

Reply:
:+1: